matrix/matrix_class.py

```diff
@@ -188,15 +195,15 @@ def get_cofactor(self, row, column):
             return self.get_minor(row, column)
         return -1 * self.get_minor(row, column)
 
-    def minors(self):
+    def minors(self) -> "Matrix":
         return Matrix(
             [
                 [self.get_minor(row, column) for column in range(self.num_columns)]
                 for row in range(self.num_rows)
             ]
         )
 
-    def cofactors(self):
+    def cofactors(self) -> "Matrix":

```

Comment:
```suggestion
    def cofactors(self) -> Matrix:

```
This would go better